install-steps.md

```diff
@@ -511,6 +511,35 @@ sudo cp ./oc /usr/local/bin/oc
 oc adm release extract --registry-config "${pullsecret_file}" --command=$cmd --to "${extract_dir}" ${RELEASE_IMAGE}
 ~~~
 
+## Retrieving the OpenShift Installer (GA Release)
+
+The `latest-4.x` (i.e. `latest-4.3`) may be used to deploy the latest 
+Generally Available version of Red Hat OpenShift Platform.
+
+Configure VARS
+    ~~~sh
+    export VERSION=latest-4.3
+    export RELEASE_IMAGE=$(curl -s https://mirror.openshift.com/pub/openshift-v4/clients/ocp/$VERSION/release.txt | grep 'Pull From: quay.io' | awk -F ' ' '{print $3}' | xargs)
```

Comment:
No need for grep or xargs:
```
curl -s https://mirror.openshift.com/pub/openshift-v4/clients/ocp/$VERSION/release.txt | awk '/Pull From: quay.io/ { print $3 }'
```

Reply:
xargs with no args is and echo - basically this is doing white space removal.

Reply:
I think it still not needed:

```
$ curl -s https://mirror.openshift.com/pub/openshift-v4/clients/ocp/latest-4.3/release.txt | awk '/Pull From: quay.io/ { print $3 }'
quay.io/openshift-release-dev/ocp-release@sha256:3a516480dfd68e0f87f702b4d7bdd6f6a0acfdac5cd2e9767b838ceede34d70d
```
```
$ curl -s https://mirror.openshift.com/pub/openshift-v4/clients/ocp/latest-4.3/release.txt | grep 'Pull From: quay.io' | awk -F ' ' '{print $3}' | xargs
quay.io/openshift-release-dev/ocp-release@sha256:3a516480dfd68e0f87f702b4d7bdd6f6a0acfdac5cd2e9767b838ceede34d70d
```

Reply:
Can we just leave as is? I don't see harm in leaving it in the expression.